src/components/OfflineWithFeedback.js

```diff
@@ -83,11 +81,6 @@ const OfflineWithFeedback = (props) => {
     const needsStrikeThrough = props.network.isOffline && props.pendingAction === 'delete';
     const hideChildren = !props.network.isOffline && props.pendingAction === 'delete' && !hasErrors;
     let children = props.children;
-    const sortedErrors = _.chain(props.errors)
-        .keys()
-        .sortBy()
-        .map(key => props.errors[key])
-        .value();
```

Comment:
I've completely removed this from above - I don't think it's beneficial - it always returns us an array (since that' what `map` returns) and trying to get it to give us a sorted object is kind of a pain. 

I'm not sure it will really ever matter that we'd have more than 1 error that needs to be sorted or that were out of order but let me know if you disagree. 

here's some testing I did 
<img width="1511" alt="image" src="https://user-images.githubusercontent.com/4073354/187732706-e3a6c083-dff0-4204-aa92-07fc8b39f917.png">
<img width="1477" alt="image" src="https://user-images.githubusercontent.com/4073354/187732782-f0bef75f-5f94-46a6-a172-a98df6492076.png">

Reply:
nvm added it back in https://expensify.slack.com/archives/C02HWMSMZEC/p1661964441330209?thread_ts=1661943704.569879&cid=C02HWMSMZEC